### ISSUE TITLE
Open source license

### ISSUE THREAD

**Issue by jcoliz** [robert-hh/SI7021#1]
Hi, would you consider adding a license to your repo? I would like to use your code in a sample I'm writing, but can only do so with an open source license.  Thanks so much!

https://help.github.com/en/articles/licensing-a-repository

"Public repositories on GitHub are often used to share open source software. For your repository to truly be open source, you'll need to license it so that others are free to use, change, and distribute the software."

**Comment by robert-hh** [comment]
I added a license note to the files. Mostly it is MIT.
I did not add myself as copyright owner, because the driver is compiled from various 
sources, so mostly the compilation, testing and writing the docs is my contribution.
The calculation of dew_point and humid_ambient are taken from the Pycom
SI7006-A20 library. Thanks to the Author. For these methods, their license applies. See:
https://github.com/pycom/pycom-libraries/tree/master/license
If that is inappropriate for your use case, just remove these two methods. They are not essential.
